src/vstart.sh

```diff
@@ -1197,10 +1203,32 @@ start_cephexporter() {
 
 do_balance_cpu() {
     local osd=$1
+    local alienstore_idx=$(( osd + CEPH_NUM_OSD ))
+
+    local reactor_interval=${cpu_table[${osd}]}
+    if ! [ "${reactor_interval}" == "" ]; then
+        local cmd="$CEPH_BIN/ceph -c $conf_fn config set osd.$osd crimson_seastar_cpu_cores ${reactor_interval}"
+        echo $cmd
+        $cmd
+    else
+        echo "No cpu_table entry for osd $osd, setting crimson_seastar_num_reactors"
+        local cmd="$CEPH_BIN/ceph -c $conf_fn config set osd.$osd crimson_seastar_num_threads $crimson_smp"
+        echo $cmd
+        $cmd
+        return
```

Comment:
If we are using `do_balance_cpu` it means that crimson_balance_cpu was set.
Letting a single OSD use `crimson_smp` without taking the `reactor_interval` will ruin the purpose of balancing cpus IIUC.
My main concern is, for example, a parsing error will result in few osds using crimson_smp and other using the balanced reactor_interval.
I think that is more consistent and less undefined behavior prone to crash here.
Alternatively fallback to an all crimson_smp allocation - but I do lean towards aborting here (as these are vstart clusters)

Reply:
The fallback option (crimson_seastar_num_threads) is used and we, for now, assume the script is woirking well for all osds.

discussed offline